src/app/pages/BorrowPage/components/AddCollateralModal/AddCollateralModal.tsx

```diff
@@ -83,14 +70,19 @@ export const AddCollateralModal: React.FC<AddCollateralModalProps> = ({
     [item.collateral, principalAsCollateral],
   );
 
+  const maintenanceRatio = useMemo(
+    () => bignumber(fromWei(item.maintenanceMargin)).add(100).toFixed(0),
```

Comment:
I mean, the number is correct but I don't think the way how we got to him is correct as well :D 

I would expect that if you have some percentage value as number somewhere, you should multiply it by 100 and not add 100 to it and I think that our `maintenanceMargin` is incorrect and it should be `1.5` and not `0.15` (because that would give us `15%` instead of `150%` we want). But let me know what you think about it.

<img width="709" alt="image" src="https://user-images.githubusercontent.com/14878713/164676071-c114623d-003b-4edb-a108-1d71dd8cdfce.png">